airbyte-integrations/connectors/source-youtube-analytics/source_youtube_analytics/manifest.yaml

```diff
@@ -0,0 +1,1518 @@
+version: 4.3.2
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - channel_annotations_a1
+    # - create_jobs
```

Comment:
Ah, I see, you commented it out so it does not make a new job?

Reply:
I actually want to create all those jobs before the sync starts. I only commented it out because it wouldn't work since `create_jobs` is not an actual stream